packages/protocol/contracts/governance/Election.sol

```diff
@@ -590,7 +590,7 @@ contract Election is
    * @return The amount of rewards that voters for `group` are due at the end of an epoch.
    * @dev Eligible groups that have received their maximum number of votes cannot receive more.
    */
-  function getGroupEpochRewards(
+  function getGroupEpochRewardsScore(
```

Comment:
Not sure why the name change. This function returns the reward amount, not the score

Reply:
renamed it to getGroupEpochRewardsBasedOnScore - problem is that since we already have function with name getGroupEpochRewards it causes problems with typescript abis